recipes/himalaya/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.30.0"
+
+
+class HimalayaConan(ConanFile):
+    name = "himalaya"
+    license = "GPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Himalaya-Library/Himalaya"
+    description = "C++ library to calculate three-loop corrections to the Higgs boson mass in the MSSM"
+    topics = ("conan", "high-energy", "physics", "hep", "Higgs", "mass", "MSSM")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("Himalaya does not support {}".format(self.settings.compiler))
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
```

Comment:
```suggestion
    def build_requirements(self):
        self.build_requires("gfortran/10.2")

```
https://conan.io/center/gfortran

Reply:
Great, many thanks!